src/main/java/com/flowingcode/vaadin/addons/chatassistant/model/Message.java

```diff
@@ -35,6 +36,7 @@
 @SuppressWarnings("serial")
 @Getter
 @Setter
+@AllArgsConstructor
```

Comment:
Why is there a need for a **public** all args constructor? The signature of the constructor is `(String, boolea, String, String, LocalDateTime)`, which is cumbersome (that's why this class had a builder).